.github/bee.yaml

```diff
@@ -0,0 +1,36 @@
+image:
```

Comment:
At this file holding helm values, I would name it as such or put it in a directory to emphasise that. For example in `.github/helm/bee.yaml`. File `bee.yaml` can also represent other configurations, like the bee configuration file itself.

Reply:
Moved it into `beekeper` subfolder